### ISSUE TITLE
[Feature Request] ?Symbol/Math rendering.

### ISSUE THREAD

**Issue by robertlugg** [Thibaut/devdocs#750]
Some documents have embedded formula such as:
\(\frac{\gamma(x-\mu)}{\sigma}+\beta\)

It would be great to render these:

![image 1240](https://user-images.githubusercontent.com/6054540/35242219-cbf4476a-ff6d-11e7-93b8-67876eecca17.png)

I am using Firefox Quantum (57.0.4)

**Comment by j-f1** [comment]
Here’s [link to DevDocs page](http://devdocs.io/tensorflow~python/tf/nn/batch_normalization), and [the original page](https://www.tensorflow.org/api_docs/python/tf/nn/batch_normalization).

It looks like they use [MathJax](https://www.mathjax.org/) with a config of `TeX-AMS-MML_SVG`. Should we load MathJax on every page?

**Comment by simon04** [comment]
DevDocs supports MathML. See PyTorch for a working example: https://devdocs.io/pytorch/generated/torch.nn.batchnorm3d#torch.nn.BatchNorm3d

cc: @lgeiger
